<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Advanced_Stack</class>
 <widget class="QWidget" name="Advanced_Stack">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1100</width>
    <height>720</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1100</width>
    <height>720</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1100</width>
    <height>720</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QStackedWidget" name="stackedWidget">
     <widget class="AdvancedSetting" name="page_set"/>
     <widget class="otawidget" name="page_ota"/>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>AdvancedSetting</class>
   <extends>QWidget</extends>
   <header>advanced_setting/advanced_setting.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>otawidget</class>
   <extends>QWidget</extends>
   <header>ota/otawidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
